Pal/src/host/Linux-SGX/pal_linux.h

```diff
@@ -148,13 +147,11 @@ extern struct pal_enclave_state {
 
 #define PAL_ENCLAVE_INITIALIZED     0x0001ULL
 
-extern struct pal_enclave {
```

Comment:
Please explain why making this change?

Reply:
It seems that the enclave_size field is only used in one place, and is also stored in TLS.  Once you are down to a single field (base address), might as well just make it a static variable.

Reply:
Understood.